docs/tidb_http_api.md

```diff
@@ -251,6 +251,76 @@
     }
     ```
 
+    *Hint: The method to convert the Hex format key returned by TiDB API into the format recognized by [tikv-ctl](https://docs.pingcap.com/tidb/stable/tikv-control).*
+    
+    Step 1: Get the hex of the key you need. For example you could find the key by the following TiDB API.
+
+     ```shell
+        $curl http://127.0.0.1:10080/mvcc/key/test/t1/1
+        {
+        "key": "7480000000000008C65F728000000000000001",
+        "region_id": 10,
+        "value": {
+            "info": {
+                "writes": [
+                    {
+                        "start_ts": 445971968923271174,
+                        "commit_ts": 445971968923271175,
+                        "short_value": "gAACAAAAAgMIAAkAc2hpcmx5YTQi"
+                    },
+                    {
+                        "start_ts": 445971959499980803,
+                        "commit_ts": 445971959499980804,
+                        "short_value": "gAACAAAAAgMIAAkAc2hpcmx5YTQL"
+                    }
+                ]
+            }
+        }
+        }
+     ```
+
+     Step 2: Convert the hex format from tidb to escaped format with [tikv-ctl](https://docs.pingcap.com/tidb/stable/tikv-control)
+
+     ```shell
+      ./tikv-ctl --to-escaped '7480000000000008C65F728000000000000001'
+      t\200\000\000\000\000\000\010\306_r\200\000\000\000\000\000\000\001
+     ```
+
+     Step 3: Encode the key to make it memcomparable in tikv with [tikv-ctl](https://docs.pingcap.com/tidb/stable/tikv-control)
+
+     ```
+     ./tikv-ctl --encode 't\200\000\000\000\000\000\010\306_r\200\000\000\000\000\000\000\001'
+     7480000000000008FFC65F728000000000FF0000010000000000FA
+     ```
+
+     Step 4: Convert the hex format to escaped format again since most `tikv-ctl` commands only accept keys in escaped format while the `--encode` command outputs the key in hex format.
```

Comment:
```suggestion
     Step 4: Convert the key from hex format to escaped format again since most `tikv-ctl` commands only accept keys in escaped format while the `--encode` command outputs the key in hex format.
```